buildconfig/stubs/pygame/rect.pyi

```diff
@@ -147,6 +158,12 @@ class Rect:
     ) -> bool: ...
     def collidelist(self, rect_list: Sequence[RectValue]) -> int: ...
     def collidelistall(self, rect_list: Sequence[RectValue]) -> List[int]: ...
+    def collideobjectsall(
+        self, objects: Sequence[Any], key: Optional[Callable[[Any], RectValue]] = None
+    ) -> List[Any]: ...
+    def collideobjects(
+        self, objects: Sequence[Any], key: Optional[Callable[[Any], RectValue]] = None
+    ) -> Any: ...
```

Comment:
I'd still prefer if generics are used here, instead of `Any`. The key benefit of that is that it checks for complete compatibility of the function call, as in something like
`collided = pygame.rect.collideobjects([1, 4, 5], key=lambda x: (x.lower(), x.lower(), 100, 100))`

With the generics, it would know that `x` in the lamba function *has* to be an integer, and it would correctly show a warning for calling `lower()` at analysis. Likewise, with generics, the typechecker also knows the precise type of the return value of the function. IE. it will return the type, which `objects` was a sequence of the same type

Check out the codeblock I gave in my first review :)